src/peeble/Transforms.fs

```diff
@@ -106,12 +143,24 @@ let convertSingleCaseUnion (ctx: PhpCompiler) (case: FSharpUnionCase) =
                         { Name = e.Name 
                           Type  = convertType e.FieldType } ]
           Methods = [ 
+              { PhpFun.Name = "allCases";
```

Comment:
This is not strictly necessary but makes the php code a bit easier to write and reason about